packages/nouns-contracts/contracts/governance/NounsDAOLogicV2.sol

```diff
@@ -0,0 +1,895 @@
+
+        // Save current value, if any, for inclusion in log
+        address oldPendingAdmin = pendingAdmin;
+
+        // Store pendingAdmin with value newPendingAdmin
+        pendingAdmin = newPendingAdmin;
+
+        // Emit NewPendingAdmin(oldPendingAdmin, newPendingAdmin)
+        emit NewPendingAdmin(oldPendingAdmin, newPendingAdmin);
+    }
+
+    /**
+     * @notice Accepts transfer of admin rights. msg.sender must be pendingAdmin
+     * @dev Admin function for pending admin to accept role and update admin
+     */
+    function _acceptAdmin() external {
+        // Check caller is pendingAdmin and pendingAdmin ≠ address(0)
+        require(msg.sender == pendingAdmin && msg.sender != address(0), 'NounsDAO::_acceptAdmin: pending admin only');
+
+        // Save current values for inclusion in log
+        address oldAdmin = admin;
+        address oldPendingAdmin = pendingAdmin;
+
+        // Store admin with value pendingAdmin
+        admin = pendingAdmin;
+
+        // Clear the pending value
+        pendingAdmin = address(0);
+
+        emit NewAdmin(oldAdmin, admin);
+        emit NewPendingAdmin(oldPendingAdmin, pendingAdmin);
+    }
+
+    /**
+     * @notice Changes vetoer address
+     * @dev Vetoer function for updating vetoer address
+     */
+    function _setVetoer(address newVetoer) public {
+        require(msg.sender == vetoer, 'NounsDAO::_setVetoer: vetoer only');
+
+        emit NewVetoer(vetoer, newVetoer);
+
+        vetoer = newVetoer;
+    }
+
+    /**
+     * @notice Burns veto priviledges
+     * @dev Vetoer function destroying veto power forever
+     */
+    function _burnVetoPower() public {
+        // Check caller is pendingAdmin and pendingAdmin ≠ address(0)
+        require(msg.sender == vetoer, 'NounsDAO::_burnVetoPower: vetoer only');
+
+        _setVetoer(address(0));
+    }
+
+    /**
+     * @notice Current proposal threshold using Noun Total Supply
+     * Differs from `GovernerBravo` which uses fixed amount
+     */
+    function proposalThreshold() public view returns (uint256) {
+        return bps2Uint(proposalThresholdBPS, nouns.totalSupply());
+    }
+
+    function proposalCreationBlock(Proposal storage proposal) internal view returns (uint256) {
+        if (proposal.creationBlock == 0) {
+            return proposal.startBlock - votingDelay;
+        }
+        return proposal.creationBlock;
+    }
+
+    /**
+     * @notice Quorum votes required for a specific proposal to succeed
+     * Differs from `GovernerBravo` which uses fixed amount
+     */
+    function quorumVotes(uint256 proposalId) public view returns (uint256) {
+        Proposal storage proposal = _proposals[proposalId];
+        if (proposal.totalSupply == 0) {
+            return proposal.minQuorumVotes;
+        }
+
+        return
+            dynamicQuorumVotes(
+                proposal.againstVotes,
+                proposal.totalSupply,
+                getDynamicQuorumParamsAt(proposalCreationBlock(proposal))
```

Comment:
You can use `proposal.creationBlock` here as `proposal.creationBlock == 0` will never be true.

Reply:
nice catch!
fixed: https://github.com/nounsDAO/nouns-monorepo/pull/438/commits/38c4cc56b60954e0e32d50b2c4acebdc4717b75d